src/components/Editor/Utilities/ColorParser.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <div><span v-html="parsedText" /> <code v-if="showRaw">({{ text }})</code></div>
+</template>
+
+<script>
+export default {
+  props: ['text', 'showRaw'],
+  created() {
```

Comment:
Since you're modifying the DOM here, you need to use `mounted()` instead of `created()`
```suggestion
  mounted() {
```